.github/workflows/aks.yaml

```diff
@@ -55,12 +55,7 @@ jobs:
       rancher_version: ${{ inputs.rancher_version || '2.8-head' }}
       k3s_version: ${{ inputs.k3s_version || 'v1.27.9+k3s1' }}
       operator_nightly_chart: ${{ inputs.operator_nightly_chart == true || (github.event_name == 'schedule' && true) }}
-      run_p0_provisioning_tests: ${{ contains(inputs.tests_to_run, 'p0_provisioning') || (github.event_name == 'schedule' && true) }}
-      run_p0_importing_tests: ${{ contains(inputs.tests_to_run, 'p0_importing') || (github.event_name == 'schedule' && true) }}
-      run_support_matrix_provisioning_tests: ${{ contains(inputs.tests_to_run, 'support_matrix_provisioning') || (github.event_name == 'schedule' && false) }}
-      run_support_matrix_importing_tests: ${{ contains(inputs.tests_to_run, 'support_matrix_importing') || (github.event_name == 'schedule' && false) }}
-      run_k8s_chart_support_provisioning_tests: ${{ contains(inputs.tests_to_run, 'k8s_chart_support_provisioning') || (github.event_name == 'schedule' && false) }}
-      run_k8s_chart_support_importing_tests: ${{ contains(inputs.tests_to_run, 'k8s_chart_support_importing') || (github.event_name == 'schedule' && false) }}
+      tests_to_run: ${{ inputs.tests_to_run || 'p0_provisioning/p0_importing' }}
```

Comment:
Will scheduled jobs be affected and continue to run ?

Reply:
They will continue to run, the default value `p0_provisioning/p0_importing` is passed, so it will be fine.